lbz/authz/wrappers.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+"""
+Authorization module.
+"""
+from functools import wraps
+
+from typing import Callable, Union, Type, Any
+
+from lbz.exceptions import PermissionDenied, Unauthorized
+from lbz.resource import Resource
+from lbz.authz.collector import authz_collector
+from lbz.authz.authorizer import Authorizer
+
+
+def check_permission(resource: Union[Type[Resource], Resource], permission_name: str) -> dict:
```

Comment:
You changed the type of resource to the wrong one. It should be equal to `resource: Resource`.